src/main/kotlin/com/exactpro/th2/act/core/messages/MessageBuilder.kt

```diff
@@ -66,17 +69,22 @@ fun main() {
     }
 }
 
-fun message(type: String, connectionID: ConnectionID, setup: MessageBuilder.() -> Unit): Message =
-    MessageBuilder(type, connectionID).also(setup).build()
+fun message(type: String, connectionID: ConnectionID, bookName: String, setup: MessageBuilder.() -> Unit): Message =
```

Comment:
Message id doesn't matter when we talk about message sent by act because the message has not been stored into cradle yet, also a conn sets own message id when it sends raw message into mstore.
Maybe use dummy book name instead of forcing user to specify useless value.
I'm not sure about this idea, @OptimumOpium what do you think about that?

Reply:
If we won't somehow allow the user to control the book name we can add a parameter as @isengrims did.
But I think we don't want that. So, it is better to remove the parameter and create the message without book name (or dummy book name)